### PR TITLE
Styles as submodule

### PR DESCRIPTION
**Pull request by Drapegnik**
- add `babajka-markup` as submodule
- compile & bundle styles on frontend
- import `.scss` per component
- [storybook](https://babajka.github.io/babajka-frontend)
- deployed to [dev.wir.by](http://dev.wir.by)
- add `deploy-from-local` script

**possible issues**:
- css bundle size increase: `209 KB` -> `262 KB` (seems like styles not minified)

### CODE REVIEW

**Comment by Drapegnik** [comment]
https://spectrum.chat/next-js/general/production-build-css-files-are-not-minified~9f9f43b8-ec8b-45e5-a8e3-5b57a62e9e67

> Mine aren't minified either. That said, CSS doesn't minify all that well and about all you'd get out of it is whitespace removal - which is essentially irrelevant with gzip/br compression anyway.

**Comment by Drapegnik** [comment]
[Markup as submodule](https://trello.com/c/w8TQ8Wpy/212-markup-as-submodule)

File: bin/deploy-dev.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin bash
```
Comment (UladBohdan) [inline bin/deploy-dev.sh]: how about `postdeploy-dev.sh` name?
I agree this script does not do any deployment

File: package.json [hunk 2]
```diff
@@ -38,8 +36,9 @@
     "@storybook/addon-knobs": "^4.1.6",
     "@storybook/addon-links": "^4.1.6",
     "@storybook/react": "^4.1.6",
+    "@zeit/next-bundle-analyzer": "^0.1.1",
+    "@zeit/next-sass": "^1.0.1",
     "Dante2": "git+https://github.com/babajka/dante2.git",
-    "babajka-markup": "^1.0.12",
```
Comment (UladBohdan) [inline package.json]: it's gone 😭

File: package.json [hunk 3]
```diff
@@ -26,7 +24,7 @@
     "analyze:server": "BUNDLE_ANALYZE=server npm run build",
     "analyze:client": "BUNDLE_ANALYZE=browser npm run build",
     "fetch-data": "npm run copy-static && npm run update-dict && npm run update-team",
-    "postinstall": "patch-package && npm run fetch-data",
+    "postinstall": "git submodule update --init && npm i --prefix styles && npm run fetch-data",
```
Comment (UladBohdan) [inline package.json:28]: so are we installing all dependencies here or only production dependencies?

**Review by UladBohdan — DISMISSED** [review]
Please don't force-push if you make any changes🙏

I verified:
* it starts locally fine.
* it deploys from locally. I'm in a cafe with poor internet connection: it took ssh 27.5MB to push Frontend and it took about 20-30 minutes of my time.

**Comment by UladBohdan** [comment]
and I'm trying to enable gzip compression right now.

File: package.json (hunk 3 above)
Comment (Drapegnik) [reply package.json:28]: good point, imho production should be enough
`npm i --prefix styles --production`

**Review by UladBohdan — APPROVED** [review]
I did:
* implement `npm run update-styles` fetching the latest styles
* verify it deploys from locally to dev server
* verify it runs locally
* enable **gzip compression**. Bundle CSS size is about 130 KB now.

I did not:
* find a way to run different `postinstall` scripts locally and on CI (to install all submodule dependencies locally and only `--production` on CI). So far we're always installing all of them

**Comment by UladBohdan** [comment]
[to enable gzip compression and caching on nginx](https://trello.com/c/H4stqbG3/328-to-enable-gzip-compression-and-caching-on-nginx)

**Comment by Drapegnik** [comment]
after gzip:
- old approach - `33.1 KB`
- new approach - `36.4 KB`

looks good

**Pull request merged**